requirements.txt

```diff
@@ -1,3 +1,4 @@
+--extra-index-url https://download.pytorch.org/whl/cu117/index.html
```

Comment:
Add a comment: "Use this to run/publish tutorials against the latest binaries during the RC stage. Comment out after the release. Each release verify the correct cuda version."